components/brave_wallet/browser/brave_wallet_utils.cc

```diff
@@ -1686,4 +1686,18 @@ GURL GetActiveEndpointUrl(const mojom::NetworkInfo& chain) {
   return GURL();
 }
 
+std::vector<std::string> GetSupportedKeyrings() {
+  std::vector<std::string> ids = {mojom::kDefaultKeyringId};
+  if (IsFilecoinEnabled()) {
+    ids.push_back(mojom::kFilecoinKeyringId);
+    ids.push_back(mojom::kFilecoinTestnetKeyringId);
+  }
+  if (IsSolanaEnabled()) {
+    ids.push_back(mojom::kSolanaKeyringId);
+  }
+
+  DCHECK_GT(ids.size(), 0u);
+  return ids;
+}
+
```

Comment:
This was used in wallet handler for account list retrieval. Should be also reused by android somehow, but not in this PR